### DISCUSSION TITLE
OGC API Processes - status codes and error messages

### DISCUSSION THREAD

**Discussion by JakobMiksch** [geopython/pygeoapi#1035] [Q&A]
I am creating a process and I am validating the input of the user. When there is a problem with the input, I would like to respond with a error message and a custom status code like 401. Currently I do not find a solution to return any other status code than 200.

Is it possible to return other status codes than 200?

**Comment by totycro** [comment]
I have the same issue, it would be great if there was e.g. some kind of exception like a BadRequestError.

Not sure if this is related, but the status code handling could also be improved for other endpoints, for example `get_jobs()` would cause an internal server error if called with a job_id that doesn't exist.
